### DISCUSSION TITLE
How to cite?

### DISCUSSION THREAD

**Discussion by mrtnrb** [PyEllips/pyElli#174] [General]
Dear developers,

thanks for the work on pyelli, I think it is a great project. It helps me a lot in several projects. Not only ellipsometry related, but also for questions concerning for instance IR reflection spectroscopy  (IRRAS, ATR-IR).

Hence the question: How do you want to be cited when we publish results that we used pyelli for? Is there (or will there be) a paper or just citing the link to the github page? I guess some kind of citation to the https://github.com/sbyrnes321/tmm and https://github.com/Berreman4x4/Berreman4x4 would also be due. 

Maybe you could add information on how you want to be cited in the documentation or readme.

Best
Martin

**Comment by domna** [comment]
Hello Martin,

thank you for reaching out on this. I'll add a citation section in the README. In the past we just cited pyElli with its zenodo doi like this: `Müller, M. J.; Dobener, F. pyElli/Zenodo, 2021;  DOI: 10.5281/zenodo.5702470` (see [this paper](https://pubs.acs.org/doi/full/10.1021/acs.jpcc.1c10107?casa_token=VBWpImOF7_0AAAAA%3AiTZ5dZPu3lyQwy1ZGbrNr4KIBKJ0gCzlkLrVclVQa8c3nlZ7-wDQ_d1vyaKiCESFjWlMctVu3t0zuXw) by @MarJMue for reference). 

The best method would be to refer to the specific version you used, i.e., use the following for Version 0.19.0: `Müller, M. J.; Dobener, F. pyElli/Zenodo , 10.5281/zenodo.10685332
Feb 20, 2024`. You can find a list of zenodo dois for the specific versions on pyElli's zenodo page: https://doi.org/10.5281/zenodo.5702469
We are also preparing a JOSS paper which you can cite eventually. But it will still take some time until it is ready.

I think it would be good if you'd also cite Berreman4x4 and tmm. However, for Berreman4x4 I don't know how they want to be cited and we could not reach the developers in the past. tmm has an arxiv paper you can cite, I think.
